fastlane/Fastfile

```diff
@@ -61,6 +61,52 @@ lane :github_release do |options|
   )
 end
 
+desc "Update dependencies to latest GitHub releases and opens a PR"
+lane :open_pr_upgrading_dependencies do |options|
+  dry_run = options[:dry_run]
+  
+  new_versions = update_native_dependencies_to_latest
+  
+  repo_status = sh("git status --porcelain")
+
+  repo_clean = repo_status.empty?
+
+  if repo_clean
+    UI.success('Git status is clean, nothing to update! 💪')
+    next
+  end
+
+  new_ios_version = new_versions[:new_ios_version]
+  new_android_version = new_versions[:new_android_version]
+
+  if dry_run
+    UI.message("ℹ️  Nothing more to do since dry_run: true.")
+    UI.message("ℹ️  New iOS dependency version is #{new_ios_version}")
+    UI.message("ℹ️  New Android dependency version is #{new_android_version}")
+    reset_git_repo
```

Comment:
Since you added this, I suggest we check that the git status is clean before starting this lane to make sure we don't reset anything accidentally.